src/components/MovieShowPage/MovieShowPage.js

```diff
@@ -66,9 +70,14 @@ class MovieShowPage extends Component {
       </label>
     }
 
+    if (this.state.movie.backdrop_path && !this.state.movie.backdrop_path.includes('NoPhotoAvailable')){
+      movieBackdrop = this.state.movie.backdrop_path
+      movieBackdropAlt = this.state.movie.title + ' backdrop'
+    }
+
     return (
       <section className='movie-show-page'>
-        <img className='background' src={this.state.movie.backdrop_path} alt={this.state.movie.title + 'backdrop'}/>
+        <img className='background' src={movieBackdrop} alt={movieBackdropAlt}/>
```

Comment:
This was a great way to handle any images that may be missing or not rendering properly.